index.md

```diff
@@ -16,98 +16,91 @@ order: 1
 
 ## What to Expect from Evaluations Tools
 {:#Expect}
 
-Web accessibility evaluation tools can help you quickly identify potential accessibility issues. You can use them through all phases of the web design and development process. Tools can provide fully-automated checks, as well as help you with manual review.
+Web accessibility evaluation tools can help you quickly identify potential accessibility issues. You can use them through all phases of the web design and development process. Tools provide fully-automated checks, as well as help you with manual review.
 
 But we cannot check all accessibility aspects automatically. Human judgement is often required. Sometimes evaluation tools can produce false or misleading results. Web accessibility evaluation tools can not *determine* accessibility, they can only *assist* in doing so.
 
 ## Features of Evaluation Tools
 {:#Features}
 
-Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Tools tend to offer different features and functionality which may help users to compare and assess Web accessibility evaluation tools for their specific needs.
+Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Tools tend to offer different features and functionality which may help users to compare and assess web accessibility evaluation tools for their specific needs.
 
-The following features describe tools found on the [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/). 
-
-### Features You Can Use as a Filter
-
-You may use the following features to reduce the number of evaluation tools listed.
+The following describes tool features found on the [Web Accessibility Evaluation Tools List](https://www.w3.org/WAI/ER/tools/). 
 
 * **Guidelines**  
 Different organizations and governments may require conformance with different accessibility standards, thus different tools support these standards. The W3C Web Content Accessibility Guidelines (WCAG) is internationally recognized as the standard for web accessibility, and supported by most tools.
 
 * **Language**  
-Evaluation tools support different languages. This includes the user interface of the tools, as well as the language of the content they support (for example for detecting language-related barriers).
+The support for different languages provided by evaluation tools. This includes the user interface of the tools, as well as the language of the content they support (for example for detecting language-related barriers).
 
 * **Type of Tool**  
-Lists the platform and type of tools that are available, e.g. authoring tool or browser plugin, command line, desktop or mobile application.
```

Comment:
Again, was written specifically to avoid passive voice. I feel strongly this should remain.

Reply:
Feel free to rewrite in active voice. The issue in my view is that the initial wording provided no explanation.